### PR TITLE
added existing contact bank case

### PR DESCRIPTION
**Pull request by y-golde**
Issue was that the validation for existing contact bank didn't take into account existing connected contacts from the current event.

added an exception to that

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionDialog.tsx [hunk 1]
```diff
@@ -43,14 +43,19 @@ const InteractionDialog = (props: Props) => {
     const { alertWarning, alertError } = useCustomSwal();
 
     const getEventContactIds = () => {
+        const currentInteractionsContactsIds = interactionData?.contacts.map(contact => contact.identificationNumber);
         const ids = interactions
             .map(interaction => {
+                console.log(interaction.contacts);
```
Comment (barnahum8) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionDialog.tsx:49]: remove

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionDialog.tsx [hunk 2]
```diff
@@ -43,14 +43,19 @@ const InteractionDialog = (props: Props) => {
...
                 if(interaction.id !== interactionData?.id){
                     return interaction.contacts
                 }
             }).flat().map((contact) => {
                 if(contact) {
-                    return contact[InteractionEventContactFields.IDENTIFICATION_NUMBER]
+                    const id = contact[InteractionEventContactFields.IDENTIFICATION_NUMBER];
+                    if (currentInteractionsContactsIds?.indexOf(id) === -1) {
+                        return id
```
Comment (barnahum8) [inline client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionDialog.tsx:57]: ;

**Review by barnahum8 — APPROVED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionDialog.tsx (hunk 1 above)
Comment (y-golde) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionDialog.tsx]: fixed

File: client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionDialog.tsx (hunk 2 above)
Comment (y-golde) [reply client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionDialog.tsx]: fixed

**Review by chenFriedman — APPROVED** [review]

**Pull request merged**